docs/DeveloperGuide.md

```diff
@@ -22,21 +22,21 @@ title: Developer Guide
 
 --------------------------------------------------------------------------------------------------------------------
 
-## Introduction 
+## **Introduction** 
 
 GuestBook is a **desktop app for managing guests in a hotel,
 optimized for use via a Command Line Interface** (CLI)
 while still having the benefits of a Graphical User Interface (GUI).
-GuestBook can get your
-guest management tasks done faster than traditional GUI apps with our efficient
-management workflow.
 The primary target audience of GuestBook is hotel managers of small hotels
```

Comment:
Should this be "are" rather than "in"?

Reply:
Should we maybe phrase it as "The primary target audience of GuestBook are hotel mangers of small hotels, such as budget hotels and backpacker's inns, who can type fast"? I am thinking would be better idea to subset backpacker's inns under a form of hotel so that it is not so confusing, if that sounds good!

Reply:
Agreed!